src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_SmartParkAndRideScreen.tsx

```diff
@@ -105,11 +107,52 @@ const Profile_SmartParkAndRideScreenContent = () => {
             <ThemeText>{t(SmartParkAndRideTexts.add.max)}</ThemeText>
           </View>
         )}
+
+        <HowItWorksSection />
       </View>
     </FullScreenView>
   );
 };
 
+const HowItWorksSection = () => {
+  const {t} = useTranslation();
+  const navigation = useNavigation<RootNavigationProps>();
+  const styles = useStyles();
+
+  return (
+    <>
+      <ContentHeading text={t(SmartParkAndRideTexts.howItWorks.heading)} />
```

Comment:
I'm not sure since the screenshot is _Coming..._ 😄, but I think this will work better if this `ContentHeading` is put in the parent component so that the `gap` is applied consistently.